backend/app/app.js

```diff
@@ -1,3 +1,7 @@
+/* This file is what sets up some of the necessary packages and 
+then directs the backend to the routes file. Routes is where 
+information can be accessed from. */
```

Comment:
app is actually the definition of the web server, and related services that you want to mount to the server

even routes can be considered an additional 'service' since under the hood, that is how Express works. (See `app.use` )